src/views/GroupTable/components/UsersTable/UsersTable.tsx

```diff
@@ -1,97 +1,90 @@
-import React from 'react';
+import React, { FC } from 'react';
 import clsx from 'clsx';
-import PropTypes from 'prop-types';
 import PerfectScrollbar from 'react-perfect-scrollbar';
 import { makeStyles } from '@material-ui/styles';
 import {
-  Card,
-  CardContent,
-  Avatar,
-  Table,
-  TableBody,
-  TableCell,
-  TableHead,
-  TableRow,
-  Typography,
-  Theme
+	Card,
+	CardContent,
+	Avatar,
+	Table,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableRow,
+	Typography,
+	Theme
 } from '@material-ui/core';
 import getInitials from '../../../../services/utils/getInitials';
+import { TableRow as ITableRow } from '../../../../store/groups/types';
```

Comment:
I prefer not using the "I" notation for interfaces. it is coming from .Net, but no real reason to keep this style. 
We can use TableRow as it is, it is an interface, and we probably won't have a class for it - so the I is redundant. what do you think?

Reply:
I agree! 
In this case I used the I notation because we had duplicate TableRow var (collides with line 13), couldn't find a better name.
I will make sure to remove the I notation from the IProps as well.

Reply:
Moved the I notation to the end, better? :)